app/models/repository.rb

```diff
@@ -10,6 +12,9 @@ class Repository < ActiveRecord::Base
   belongs_to :parent,   class_name: "Repository", foreign_key: "make_id"
   paginates_per 12
 
+  # default_scope { where(:share_type => "public") }
+  scope :public_repos, -> { where(share_type:"public") }
```

Comment:
Maybe? I think the question comes down to which is the better default.

If you did `Repository.all` should you get private repos? On the other hand, if you do `@user.repositories` should you _not_ get private repos?